pkgs/development/libraries/ffmpeg/generic.nix

```
@@ -47,6 +47,7 @@
 , withGme ? withFullDeps # Game Music Emulator
 , withGnutls ? withHeadlessDeps
 , withGsm ? withFullDeps # GSM de/encoder
+, withHarfbuzz ? withHeadlessDeps # Needed for drawtext filter
```

harfbuzz support was only added in 6.1

```suggestion
, withHarfbuzz ? withHeadlessDeps && lib.versionAtLeast version "6.1" # Needed for drawtext filter
```